docs/config-file-ort-yml.md

```diff
@@ -125,11 +126,10 @@ scopes defined in the examples below match the scopes in your project.
 
 ### When to Use Curations
 
-Project-specific curations should be used when you want to correct the licenses detected in the source code of the
-project. If you need to correct the license findings for a third-party dependency then add a curation to
-[curations.yml](config-file-curations-yml.md) or [package configuration](config-file-package-configuration-yml.md).
+Project-specific license finding curations should be used when you want to correct the licenses detected in the source 
+code of the project.
 
-### Curating License Findings
+### Curating Project License Findings
```

Comment:
Why add `Project`? License finding curations apply also to `Packages`.

Reply:
The documentation states:
> Project-specific curations should be used when you want to correct the licenses detected in the source code of the
project.

Which implicitly excludes packages.
The [used Model](https://github.com/oss-review-toolkit/ort/blob/e9204dc54768b07967873d7ecd3506d5a4c1a1dc/model/src/main/kotlin/config/LicenseFindingCuration.kt#L33-L78) doesn't define an `Identifier`.
How can the described `license_finding_curations` in this case apply to packages?

Reply:
You're right, my bad.